web/popminer/api.go

```diff
@@ -24,8 +24,24 @@ const (
 	MethodBitcoinUTXOs   Method = "bitcoinUTXOs"   // Retrieve bitcoin UTXOs
 )
 
+// ErrorCode is a unique identifier used to differentiate between error types.
+type ErrorCode string
```

Comment:
I don't think we should use strings as the key. I feel like an int with a textual representation is a better option. In my experience string compares as error values tends to end in tears.